src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.html

```diff
@@ -1 +1,27 @@
-<p>vendor-filter works!</p>
+
+<p style="font-size: 30px;" id="SIc"> Status</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 1</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 2</p>
+<mat-checkbox color="primary" class="CheckBox"></mat-checkbox>
+<p style="font-size: 15px;">status 3</p>
+
+<hr>
+<p style="font-size: 30px;">Work Types
+    <mat-form-field appearance="outline" style="width: 60vw; font-size: 12px;">
+    <input matInput>
+        <mat-chip-list aria-label="Work Type Selector" class="chips">
+            <mat-chip>words</mat-chip>
+        </mat-chip-list>
+        <mat-chip-list aria-label="Work Type Selector" class="chips">
+            <mat-chip>words</mat-chip>
+        </mat-chip-list>
+    </mat-form-field> 
+</p>
+<hr>
+<button mat-button style="font-size: 20px;" color="primary">Clear</button>
```

Comment:
We could probably add basic functionally like clear input and exiting form upon cancel and returning input on apply then closing form 👍🏻